Docs for `sentry-graphql-22` (#12259)
getsentry/sentry-docs: docs/platforms/java/common/integrations/graphql.mdx

Our GraphQL integration can be configured automatically if you're using `spring-graphql` with either the `sentry-spring-boot-starter` or the `sentry-spring-boot-jakarta-starter` integration.

<Note>

> If you are using version 22 of `graphql-java`, please refer to <PlatformLink to="/integrations/graphql22/">GraphQL 22 docs</PlatformLink>.

Reviewer: ```suggestion
If you're using version 22 of `graphql-java`, please refer to <PlatformLink to="/integrations/graphql22/">GraphQL 22 docs</PlatformLink>.
```
